pkgs/development/libraries/theft/default.nix

```diff
@@ -0,0 +1,33 @@
+{ stdenv, fetchFromGitHub }:
+
+stdenv.mkDerivation rec {
+  version = "0.4.3";
+  name = "theft-${version}";
+
+  src = fetchFromGitHub {
+    owner = "silentbicycle";
+    repo = "theft";
+    rev = "v${version}";
+    sha256 = "1ibh8np12lafnrsrvjbbzlyq45zq654939x0y22vdnc6s8dpbhw4";
+  };
+
+  preConfigure = "patchShebangs ./scripts/mk_bits_lut";
+  # doCheck = true;
```

Comment:
Any particular reason checks are not enabled? Please document that.

Reply:
checks are now enabled